docs/source/how-to/deploying-substra/index.rst

```diff
@@ -1,20 +1,27 @@
-********
-Overview
-********
+How-to guides for deploying Substra
+===================================
```

Comment:
It seems to me this section isn't just how-to guides, are you planning on splitting it up further?

Reply:
It was a first pass on trying to organise stuff, but maybe the walk-through should be moved to the tutorials indeed. 
Open to suggestion as to what we want to put where :)

Reply:
Since we're serving the needs of different people, perhaps we could add an additional layer on top of the diataxis one?

So you'd select **Persona** (DS, developer, SRE) and then **mode** (reference, tutorial, how-to, explanation)

Apologies, this is a pretty fundamental suggestion to make when you've already made all these changes

Reply:
Personally I'd rather have the **persona** under the **mode** rather than the other way. The API reference is going to be the same for all personas, and some other things may be hard to classify (some part of the explanations might be of interest to everyone, whereas other parts will interest advanced users, be they data scientists or SRE). 
What do you think?

Reply:
It's a good point that there is a lot of common from one to the other. Although we can possibly use links (or even straight copy, since we have an automated build system) to mitigate this.

I'm no expert (let's hire a technical writer!) so my opinion is just that, but I feel persona is more fundamental than mode.

In a single work session there's a good chance I'll access multiple modes, but I'll probably stick to the same persona (true for my use case, but maybe as an MLE you need to access both DS and developer personas?).

And I'd rather have a big switch at the top of the screen (a bit radical lol, but that's how openshift does it) than "if you are deploying substra" plastered the middle of the text

Reply:
Thinking about it, I'm not sure that I would consider the deployment walk-through as a tutorial, in the sense that tutorials are supposed to be focused on teaching new things. 
In this lens, the walk-through is more of a long how-to guide for deploying. 
I've arranged / renamed things a bit, let me know what you think.